### PR TITLE
Reference url metrics under their other name (`url2`) in generated views

### LINKED ISSUE #539: Expose `url` types in Glean Ping views
The `url` type was added to the Glean client and is being used to send data for MDN, but this codebase doesn't propagate those fields to the Looker views. This ticket is for exposing them.

It should start with exposing them as `ALLOWED_TYPES` here (that might be everything, but also add a test).

### PR DESCRIPTION
**Pull request by badboy**
Fixes #539

### CODE REVIEW

File: generator/views/glean_ping_view.py [hunk 1]
```diff
@@ -29,6 +29,14 @@
     "rate",
     "timespan",
     "uuid",
+    "url",
+}
+
+# Some metric types are exposed under different names (due to bugs).
```
[the remark this answers is not in the document]
Comment (fbertsch) [reply generator/views/glean_ping_view.py:35]: Oops, you can ignore this. I see the bug number below

File: generator/views/glean_ping_view.py [hunk 2]
```diff
@@ -29,6 +29,16 @@
     "rate",
     "timespan",
     "uuid",
+    "url",
```
Comment (fbertsch) [inline generator/views/glean_ping_view.py:32]: looks like we need `labeled_rate` and `text` here

Comment (badboy) [reply generator/views/glean_ping_view.py:32]: `labeled_rate` does not really exist (which means I should actually remove it from the other list, too)
`text` exist, but not currently used (afaik) and in general category 3 or 4. Not sure if we want to default-expose that.

**Comment by badboy** [comment]
ok, now I think it is ready

**Pull request merged**